### PR TITLE
bots: Update RHEL URLs, fix missing logo

### PR DESCRIPTION
**Pull request by martinpitt**
- Consistently use nightly repository for rhel-7-6.
 - Use *.devel.* instead of *.eng.bos* to auto-select closest mirror.

 * [x] image-refresh rhel-7-6
 * [x] image-refresh rhel-x

### CODE REVIEW

**Comment by martinpitt** [comment]
Resetting. Our openshift test cluster is currently AWOL due to not being able to reach fedorapeople.org. I notified e2e support.

**Comment by cockpituous** [comment]
Task aborted.

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-vn7n5.
Log: https://209.132.184.41:8493/logs/image-refresh-10052-20180912-150404/

**Comment by cockpituous** [comment]
image-refresh rhel-7-6 done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-7-6-20180912-155151

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-8rsfz.
Log: https://209.132.184.41:8493/logs/image-refresh-10052-20180912-155959/

**Comment by cockpituous** [comment]
image-refresh rhel-x done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-x-20180912-163131

**Comment by martinpitt** [comment]
All the libvirt tests fail like this:

    error: internal error: child reported: Kernel does not provide mount namespace: Permission denied

This is a regression in kernel/systemd/qemu/libvirt and needs a closer examination.

**Comment by martinpitt** [comment]
Disregard the previous comment, that was for a different PR.

**Comment by martinpitt** [comment]
[Current rhel-x test run](https://209.132.184.41:8493/logs/pull-10052-20180913-191111-a798414f-verify-rhel-x/log.html) for flake comparison.

File: tools/cockpit.spec [hunk 1]
```diff
@@ -491,6 +491,9 @@ Recommends: sscg >= 2.3
 Requires(post): systemd
 Requires(preun): systemd
 Requires(postun): systemd
+%if 0%{?rhel} && 0%{?centos} == 0
+Requires: redhat-logos
+%endif
```
Comment (martinpitt) [inline tools/cockpit.spec]: Argh! Yesterday it turned out that this should actually be an unconditional dependency on `system-logos`. So I need to break the nice green-ness again here.

Comment (martinpitt) [reply tools/cockpit.spec]: Fixed.

**Pull request merged**